### ISSUE TITLE
lazy fetching of the monero blockchain

### ISSUE THREAD

**Issue by milahu** [monero-project/monero#9399]
i assume that 90% of the blockchain are never used
because most transactions i see (event horizon) are only some weeks old
so the full monero blockchain of currently 90GB is mostly a waste of disk space

why no lazy fetching?
let me use a "shallow clone" of the blockchain
and only when a transaction requires deeper access to the blockchain
then fetch to the required depth
and maybe later delete the deep end of the chain to free disk space

**Comment by selsta** [comment]
I don't think this has a realistic chance of being implemented. It would require a lot of work regarding how this could be implemented in a secure way, and it would also increase load on the network. If someone does not have enough storage for a pruned blockchain it's best that they use a remote node.

**Comment by jayson687** [comment]
The best hackers I have seen are  d a r k w e b p r o g r a m m e r @ g m a i l . c o m   , they got me $15,000 via hack
